src/widgets/StepperRow.js

```diff
@@ -60,7 +60,7 @@ const localStylez = {
 };
```

Comment:
wait, `localStylez`?? how did that slip past?

Reply:
whoops will get that fixed up 🤣